### ISSUE TITLE
Proxmox kernel update available

### ISSUE THREAD

**Issue by naturboy** [remotephone/remotephone.github.io#1]
Hi! I just wanted to inform you that the kernel update for you docker-swarm lxc problem is now available (https://forum.proxmox.com/threads/linux-kernel-5-3-for-proxmox-ve.59398/). Some how I still can't get my setup to work. Maybe you could update your guide?

**Comment by remotephone** [comment]
Hey sorry for not seeing this, I upgraded to that kernel version, poked it a bit, and got caught up with life. Seems like I'll be trapped inside for the time being, so I'll try again and update with what I get. Thanks!

**Comment by remotephone** [comment]
@naturboy, I poked at this for quite a while a few different times and still couldn't get it to work, I ended up abandoning it and just running it on VMs in my cluster. With ram ballooning enabled, I think there's more flexibility in ram usage, but I have a hunch resources will still be held tighter than if I were using containers. Other folks I could find trying what I was trying all ran into the same issues. The swarm networks come up on the lead, but workers won't see the network and iptables won't set rules correctly. I tried inserting kernel modules manually, unconfined containers, nesting, various keyctl settings, all with no luck. i wish I had something better to say on this, sorry. 

Some references:
- https://forum.proxmox.com/threads/docker-support-in-proxmox.27474/page-6#post-295237
- https://forum.proxmox.com/threads/missing-br_netfilter-module.53791/
